lib/services.js

```diff
@@ -476,6 +476,7 @@ export default {
             params.headers['content-type'];
         multipartObjectMD.expires =
             params.headers.expires;
+        multipartObjectMD.storageClass = params.storageClass;
```

Comment:
This change was to accommodate an `undefined` value when using the file backend. @rahulreddy and I have traced this down to [this line](https://github.com/scality/Arsenal/blob/master/lib/algos/list/MPU.js#L58) in Arsenal. Here the value should be `tmp['x-amz-storage-class']`. I will make that change to handle the file backend, and a change to handle in-memory value in a new commit. Thanks!